.github/workflows/upstream-dev-ci.yaml

```diff
@@ -55,7 +62,7 @@ jobs:
   report:
     name: report
     needs: upstream-dev
-    if: "always()&&(github.event_name == 'schedule')&&(github.repository == 'pydata/xarray')"
+    if: "always()&&(github.event_name == 'schedule')&&(github.repository == 'pydata/xarray')&&(needs.upstream-dev.outputs.artifacts_availability =='true')"
```

Comment:
this is a rather long line to read and understand. Do you know if the syntax allows breaking this into multiple lines?

Reply:
The latest commit fixes this.